src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days >= 3 && days < 7) {
+    return days * 40 - 20;
+  } else if (days >= 7) {
```

Comment:
As 'days >= 7' is the only remaining condition after the previous checks, you don't need an 'else if' here. You can directly return the result. This adheres to the checklist items 4 and 5.